db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+    change_column_comment :vms, :deprecated, "HELP"
+    change_column_comment :vms, :description, "A custom description that users can add within the app."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
+    change_column_comment :vms, :ems_id, "ID of the ExtManagementSystem associated with the VM."
+    change_column_comment :vms, :evm_owner_id, "ID of VM owner from product database."
+    change_column_comment :vms, :ems_ref_obj, "HELP"
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+    change_column_comment :vms, :fault_tolerance, "HELP"
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "HELP"
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "The last date when performance metrics were captured."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState Analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "HELP"
+    change_column_comment :vms, :linked_clone, "HELP"
+    change_column_comment :vms, :location, "The location or region of the VM."
+    change_column_comment :vms, :memory_hot_add_enabled, "HELP"
+    change_column_comment :vms, :memory_hot_add_increment, "HELP"
+    change_column_comment :vms, :memory_hot_add_limit, "HELP"
+    change_column_comment :vms, :memory_limit, "HELP"
+    change_column_comment :vms, :memory_reserve, "HELP"
+    change_column_comment :vms, :memory_reserve_expand, "HELP"
+    change_column_comment :vms, :memory_shares, "HELP"
+    change_column_comment :vms, :memory_shares_level, "HELP"
+    change_column_comment :vms, :miq_group_id, "MIQ Group associated with the VM."
+    change_column_comment :vms, :name, "The name of the VM/Template."
+    change_column_comment :vms, :orchestration_stack_id, "ID of the orchestration stack associated with the VM."
+    change_column_comment :vms, :power_state, "The current power state, typically 'on' or 'off'."
+    change_column_comment :vms, :previous_state, "The previous power state change."
+    change_column_comment :vms, :publicly_available, "Indicates whether the VM is public or private."
+    change_column_comment :vms, :raw_power_state, "HELP"
+    change_column_comment :vms, :registered, "HELP"
+    change_column_comment :vms, :resource_group_id, "ID of the resource group that the VM/Template belongs to."
+    change_column_comment :vms, :retired, "Indicates whether or not the VM is retired."
+    change_column_comment :vms, :retirement_last_warn, "HELP"
+    change_column_comment :vms, :retirement_requester, "HELP"
```

Comment:
Since the actual column from the user table is `userid`, then I think "ID" might be confusing (since it's not the actual `id` column.  Perhaps "userid of user who requested retirement"?